assets/webview/pages/terminal/index.js

```diff
@@ -1,16 +1,13 @@
 import VuePage from '../../script/vue/page.js'
 import * as ResultView from './result-view/index.js'
 
-const tip = {
-  sqlite3: '* sqlite3 每次仅可执行一条 sql（输入多条 sql 将只执行第一条）'
```

Comment:
SQLite3 can only execute one SQL at a time (entering more than one SQL will ONLY execute the first one)